### PR TITLE
Added Mvg Widget

### PR DESCRIPTION
**Pull request by m42e**
[m42e](https://github.com/m42e) ✨ Feature Large [m42e /mvg-upstream → Lissy93/dashy](https://github.com/Lissy93/dashy/tree/mvg-upstream) Commits: 5 | Files Changed: 5 | Additions: 816 Unchecked Tasks

Display mvg departure times for MVG station.

**Category**: Feature

**Overview**
Display The departure times of the next public transportation for Munich.

**New Vars** _(if applicable)_

**Field** | **Type** | **Required** | **Description**
--- | --- | --- | ---
**`location`** | `string` |  Required | The name of the location (exact) or the location id, startin with `de:09162:`
**`limit`** | `integer` |  _Optional_ | Limit number of entries, defaults to 10.

`defaults.mvg`

**Screenshot**

Mvg list of departures

**Code Quality Checklist** _(Please complete)_
- [x] All changes are backwards compatible
- [x] All lint checks and tests are passing
- [x] There are no (new) build warnings or errors
- [x] _(If a new config option is added)_ Attribute is outlined in the schema and documented
- [ ] Bumps version, if new feature added ?

### CODE REVIEW

**Review by Lissy93 — APPROVED** [review]
Looks great :)

Are you ready for me to go ahead and merge?

File: src/components/Widgets/Mvg.vue [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    }
+  },
+  filters: {
+    formatDepartureTime(timestamp) {
+      const msDifference = new Date(timestamp).getTime() - new Date().getTime();
+      const diff = Math.max(0, Math.round(msDifference / 60000));
+      return diff;
+    },
+    limitLength(str) {
+      return str;
```
Comment (Lissy93) [inline src/components/Widgets/Mvg.vue:76]: Doesn't look like this is being used, same in [`MvgConnection.vue`](https://github.com/Lissy93/dashy/blob/75b06f8af80b14f44f40b92b18df994729dd53d8/src/components/Widgets/MvgConnection.vue#L93-L95)

File: src/components/Widgets/Mvg.vue [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+      if (this.options.location.startsWith('de:09162:')) return true;
+      return false;
+    },
+    offset() {
+      if (this.options.offset) return this.options.offset;
+      return 0;
+    },
+    limit() {
+      if (this.options.limit) return this.options.limit;
+      return 10;
```
Comment (Lissy93) [inline src/components/Widgets/Mvg.vue:95]: This could be shortened to just `return this.options.limit || 10`
But what you've got also works

**Comment by m42e** [comment]
Yes, you can go ahead.

**Review by Lissy93 — APPROVED** [review]

**Pull request merged**